google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/TopicSnippets.java

```diff
@@ -114,94 +114,6 @@ public boolean deleteAsync() throws ExecutionException, InterruptedException {
   }
 
   /**
-   * Example of publishing one message to the topic.
-   */
-  // [TARGET publish(Message)]
-  public String publishOneMessage() {
```

Comment:
In your tracking issue for to do items, split out sample code into multiple items:

* snippets
* examples (like this one here)